.github/workflows/add-pdf-url-to-manuscript.yaml

```diff
@@ -32,18 +32,22 @@ jobs:
           cache: yarn
       - name: yarn install
         run: yarn
-      - name: Add PDF url to manuscript
+      - name: Prepare manuscripts.json with PDF url
         run: |
           cat manuscripts.json | \
             yarn --silent add-pdf-url-to-manuscript \
             --msid "${{ github.event.inputs.msid }}" \
             --versionManuscript "${{ github.event.inputs.versionManuscript || 1 }}" \
             --urlPdf "${{ github.event.inputs.urlPdf }}" | \
-            tee manuscripts.json > /dev/null
+            tee manuscripts-new.json > /dev/null
```

Comment:
I think the issue was caused by beginning to write to the output stream of the same file that we were reading in. This is working for me.